packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js

```diff
@@ -27,6 +29,8 @@ export default class InMeetingActions {
 
   get() {
     return {
+      canInviteGuest: this.canInviteGuest,
+      canAdmitParticipant: this.canAdmitParticipant,
```

Comment:
should not matter much but should we keep the same name guest/ participant !?

Reply:
I wasn't sure they are always the same thing... Are you definitely admitting a guest? Or can invitees be admitted as well

Reply:
End of the day they all come as participant , So was thinking canInviteNewParticipants and stick with participant and not guest

Reply:
ok sure. I've changed to canInviteNewParticipants

Reply:
looks good, let me know if i need to merge , Its waiting for someone from web client to provide approval